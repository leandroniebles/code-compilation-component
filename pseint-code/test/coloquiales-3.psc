Proceso sin_titulo
	escribir es cero
	escribir x es cero z
	escribir x, es cero, z
	escribir r,x es cero z,p
	escribir es divisible por
	escribir x es divisible por z
	escribir x,es divisible por,z
	escribir r,x es divisible por z,p
	escribir es cero;
	escribir x es cero z;
	escribir x, es cero, z;
	escribir r,x es cero z,p;
	escribir es divisible por;
	escribir x es divisible por z;
	escribir x,es divisible por,z;
	escribir r,x es divisible por z,p;
FinProceso
