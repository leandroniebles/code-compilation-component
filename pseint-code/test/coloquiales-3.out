=== Line 2: SynError 317
=== Line 3: SynError 319
=== Line 4: SynError 317
=== Line 5: SynError 319
=== Line 6: SynError 317
=== Line 6: SynError 318
=== Line 8: SynError 317
=== Line 8: SynError 318
=== Line 10: SynError 317
=== Line 11: SynError 319
=== Line 12: SynError 317
=== Line 13: SynError 319
=== Line 14: SynError 317
=== Line 14: SynError 318
=== Line 16: SynError 317
=== Line 16: SynError 318
